packages/next/build/webpack/loaders/next-image-loader.js

```diff
@@ -9,14 +9,15 @@ const VALID_BLUR_EXT = ['jpeg', 'png', 'webp']
 function nextImageLoader(content) {
   const imageLoaderSpan = this.currentTraceSpan.traceChild('next-image-loader')
   return imageLoaderSpan.traceAsyncFn(async () => {
-    const isServer = loaderUtils.getOptions(this).isServer
+    const { isServer, isDev } = loaderUtils.getOptions(this)
     const context = this.rootContext
     const opts = { context, content }
     const interpolatedName = loaderUtils.interpolateName(
       this,
       '/static/image/[path][name].[hash].[ext]',
       opts
     )
+    const outputPath = '/_next' + interpolatedName
```

Comment:
This needs to consider `assetPrefix`

Reply:
I see, the `assetPrefix` is only needed for the URL, not the file system path, right?

Reply:
Yeah indeed!